CMakeLists.txt

```diff
@@ -602,6 +619,8 @@ set(CMAKE_CUDA_FLAGS_DEVICEDEBUG "${QUDA_NVCC_FLAGS} -G" CACHE STRING
   "Flags used by the C++ compiler during device-debug builds.")
 set(CMAKE_CUDA_FLAGS_DEBUG "${QUDA_NVCC_FLAGS} -g -DHOST_DEBUG -G" CACHE STRING
   "Flags used by the C++ compiler during full (host+device) debug builds." )
+set(CMAKE_CUDA_FLAGS_SANITIZE "${QUDA_NVCC_FLAGS} -g -lineinfo -DHOST_DEBUG -Xcompiler -fsanitize=address,-fsanitize=undefined" CACHE STRING
```

Comment:
Shouldn't the host compiler flags be propagated from the CXX_FLAGS here, like we do for the other flags?

Reply:
Yes that's true.  If the propagate option is disable though then it wouldn't be.  Remind me, what was the rationale in having the option to not propagate?

Reply:
Not sure. Might be a legacy thing as the old _FindCUDA_ module before native CUDA support had that. Only other reason I can think of right now is if one CXX flag causes trouble with CUDA compilation to disable it and have the option to set it manually.

Reply:
So what do you think I should do here?  I'm happy to remove the explicit passing of the sanitize flags since they'll be propagated by default anyway.